net-analyzer/nuttcp/files/nuttcp-8.1.4-makefile.patch

```diff
@@ -0,0 +1,13 @@
+diff -r 7a32184b8e06 Makefile
+--- a/Makefile	Thu Sep 19 18:24:30 2013 +0300
++++ b/Makefile	Thu Sep 19 18:26:01 2013 +0300
+@@ -2,8 +2,7 @@
+ #EXTRAVERSION=-pre1
+ CC = gcc -Wall
```

Comment:
It seems that the Makefile doesn't honor "CC" and "LDFLAGS".

Reply:
Yes, you are right. Fixed.